pos_mail_receipt/static/src/js/screens.js

```diff
@@ -48,9 +48,16 @@ odoo.define("pos_mail_receipt.screens", function (require) {
             var self = this;
             options = options || {};
             var timeout = typeof options.timeout === 'number' ? options.timeout : 7500;
-            this.$('.button.email').addClass("highlight");
+            var timeout_db_sync = typeof options.timeout_db_sync === 'number' ? options.timeout : 5000;
```

Comment:
```suggestion
            var timeout_db_sync = typeof options.timeout_db_sync === 'number' ? options.timeout_db_sync : 5000;
```

Alternatively, just use the same `timeout` as the line above it.

Reply:
done, you're right, it's simpler this way.